### ISSUE TITLE
Universe list crashes when never created any universes

### ISSUE THREAD

**Issue by phxql** [vlow/slingring#1]
```
[moe@laptop slingring]$ universe list
Traceback (most recent call last):
  File "/usr/bin/universe", line 78, in <module>
    main()
  File "/usr/bin/universe", line 31, in main
    process_cli_arguments()
  File "/usr/bin/universe", line 74, in process_cli_arguments
    args.func(args)
  File "/usr/share/slingring/slingring-tools/universe/workflow/listing.py", line 34, in list_universes_by_args
    list_universes(args.verbose)
  File "/usr/share/slingring/slingring-tools/universe/workflow/listing.py", line 42, in list_universes
    multiverse_directory = os.listdir(multiverse_directory_path)
FileNotFoundError: [Errno 2] No such file or directory: '/home/moe/.slingring/multiverse'
```

**Comment by phxql** [comment]
I didn't create any universe before.

**Comment by vlow** [comment]
Can reproduce. Fixed in 7999116.
